pkg/trace/api/otlp_test.go

```diff
@@ -546,7 +546,7 @@ func TestOTLPHelpers(t *testing.T) {
 	})
 
 	t.Run("status2Error", func(t *testing.T) {
-		for _, tt := range []struct {
+		for _, tt := range []*struct {
```

Comment:
I don't understand why this change is needed/how it is related to gogo protobuf, could you elaborate?

Reply:
Because the new protobuf definitions incorporate a mutex; mutexes shouldn't be copied. So either we use pointers, or we make "manual" copies that don't copy the mutex. I've actually done that in a couple of places where copies seemed necessary, but mostly just switched to references instead of copies. In any case I was waiting for @knusbaum review to better understand if the changes are OK.

Reply:
Also, we still have to fix the CircleCI integration tests. Looking into it.

Reply:
Okay, thanks for the explanation! Maybe you can mark as draft until the test suite is fixed and things are reviewed by @knusbaum? (So that we know when to review)

Reply:
It's fixed.